wp-content/themes/twentyseventeen-wp20/style.css

```
@@ -1760,6 +1768,8 @@ iframe.wp-embedded-content {
 
 .wp20-poetry {
 	flex: 1;
+	position: relative;
+	top: 2px;
 	text-align: center;
```

It looks like the image just isn't being centered within this div correctly, prefer to use flex to ensure this:
```suggestion
	display: flex;
	justify-content: center;
	align-items: center;
```

---

Actually I've got a more complete fix for this in https://github.com/WordPress/wp20.wordpress.net/pull/85 so you can potentially drop this change

---

Ah, that's a better approach, thanks 👍🏻 Reverted this one in 70e782d